dashboard/common/utils.py

```diff
@@ -46,6 +46,7 @@ def get_git_version_info() -> Optional[GitInfo]:
 
     return {
         "repo": format_github_url_using_https(repo),
+        "repo_abbrev": format_github_url_using_https(repo)[:-4],
```

Comment:
What's the reason for this change? 

It looks like GIT_REPO is 
`https://github.com/tl-its-umich-edu/my-learning-analytics`

And doing this
`utils.format_github_url_using_https(repo)[:-4]`

just returns
`'https://github.com/tl-its-umich-edu/my-learning-analy'`

Which wouldn't work.

Reply:
When I was testing it the GIT_REPO automatically adds ".git" at the end of the link so the link was broken (so I got rid of the extra ".git"):

![image](https://user-images.githubusercontent.com/44738429/159094309-6432da9a-81a5-4b16-a9f0-7658b4d07252.png)

Reply:
Hmm, that sounds like something the method `format_github_url_using_https` _should_ be removing.

```
>>> repo="git@github.com:tl-its-umich-edu/my-learning-analytics.git"
>>> utils.format_github_url_using_https(repo)
'https://github.com/tl-its-umich-edu/my-learning-analytics'
```

What's your GIT_REPO from the start script? 
`git config --local remote.origin.url`

We normally clone using the[ HTTPS URL](https://docs.github.com/en/get-started/getting-started-with-git/about-remote-repositories#cloning-with-https-urls) since you can just use a token and it's generally easier to work with than SSH keypairs.

Reply:
Here's what I get when running that command: https://github.com/saputran/my-learning-analytics.git (I forked it so it has my username).

Does the current way work for you when you try to use the link? I tried diffing the link it sends me to and the link it's supposed to be and get that extra ".git".

Reply:
Hmm! It does look like the git clone is adding
`https://github.com/tl-its-umich-edu/my-learning-analytics.git`

I'd say that's a different/new bug. You can file it as a new issue but don't have to fix it. I'm not sure when it was introduced. 

Mine and the one on our server has

```
origin  https://github.com/tl-its-umich-edu/my-learning-analytics (fetch)
origin  https://github.com/tl-its-umich-edu/my-learning-analytics (push)
```

I would run this to update it to the format we're using just for purposes of this bug. 

`git remote set-url origin https://github.com/saputran/my-learning-analytics`

If I was fixing it I'd fix it here
https://github.com/tl-its-umich-edu/my-learning-analytics/blob/474de8d4eb230587f826ad7d72f4a0ec80dbf687/dashboard/common/utils.py#L18

And _probably_ have it `.replace(".git", "")` on every string.

Reply:
Thanks! I've updated the code (and added the branch link) and re-committed. I'm not super familiar with git - how do I update my pull request with the updated code? I think right now it just says there are 2 commits (one of which is the updated one).

Reply:
Thanks! That's fine, typically a pull request will have multiple commits against it. When we squash and merge it it'll go down to one commit. 

If you're did have a lot you could rebase and force push. There is a pretty good guide for that here: https://docs.gitlab.com/ee/topics/git/git_rebase.html

But it's a little more advanced topic that you'll probably get very familiar with someday!

Reply:
Perfect - thank you! What else is needed to be done for this issue?

Reply:
I created #1348 for this new issue

Reply:
It looks good to me. Thanks!